packages/graphql-server/src/functions/graphql.ts

```diff
@@ -317,7 +339,7 @@ export const createGraphQLHandler = ({
 
   const createSharedEnvelop = envelop({
     plugins,
-    enableInternalTracing: isDevEnv,
+    enableInternalTracing: tracing,
```

Comment:
Having tracing be explicitly set, so can turn on in staging or prod as needed.

Argument could be made to have this be an envar for api side.